### DISCUSSION TITLE
Issue with pm 2.5

### DISCUSSION THREAD

**Discussion by mariomare22** [ajfriesen/ESPHome-AirGradient#12] [General]
Hello Andrej!
I am running the dyi since 48h but in home assistant the pm2.5 is showing as unknown and not reading values. I am using the yaml you created (thanks for that).

Do you know if there are any troubleshooting steps to follow? (The fan in the sensor is spinning)

**Comment by ajfriesen** [comment]
Just updated esphome and my device and everything works fine.
Did you check the uart change?
https://esphome.io/changelog/2023.4.0.html#uart-ids

**Reply by pott3r3r** [reply]
I have the S8 not getting any values, but PM2.5 works

**Reply by ajfriesen** [reply]
Not sure how I can help since it is working for me. I think it might make more sense to ask on the official channels for esphome or home assistant.

**Reply by pott3r3r** [reply]
OK, thnx; I prolly got a broken sensor or somewhat :)
